aopy/analysis/base.py

```diff
@@ -532,6 +532,35 @@ def calc_corr_over_elec_distance(acq_data, acq_ch, elec_pos, bins=20, method='sp
 
     return dist, corr
 
+def subtract_erp_baseline(erp, time, t0, t1):
+    '''
+    Subtract pre-trigger activity from trial-aligned data.
+
+    Args:
+        erp (nt, nch, ntr): trial-aligned evoked responses
```

Comment:
Is there a reason to restrict this to 3D? It would be more generalizable if you could enter an arbitrary number of dimensions.

Reply:
sure, but i couldn't figure out how to do that